terragrunt/openai_api_keys.tf

```
@@ -43,3 +43,32 @@ module "ai_answers_api_key" {
     }
   ]
 }
+
+module "platform_core_hackathon_api_key" {
+  source                     = "./modules/openai_api_key"
+  subscription_id            = "c4122b45-f2e3-4873-a7fe-b94c1ad2589f" # CDS-AI subscription
+  name                       = "platform_core_hackathon"
+  custom_subdomain_name      = "platform_core_hackathon"
+  resource_group_name_prefix = "platform_core_hackathon"
+  budget_amount              = 50
+  requestor_emails           = ["patrick.heard@cds-snc.ca"]
```

Could this be our team email instead?
```suggestion
  requestor_emails           = ["platform-core-services@cds-snc.ca"]
```